<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Кампания</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="passPushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Пройти</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="characterPushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Персонажи</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="backPushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QListWidget" name="mapsListWidget"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="technologiesPushButton">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Технологии</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Система</string>
    </property>
    <addaction name="settingsAppAction"/>
    <addaction name="backAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="aboutAppAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Аккаунт</string>
    </property>
    <addaction name="aboutAccountAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aboutAppAction">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Выйти из аккаунта</string>
   </property>
  </action>
  <action name="backAction">
   <property name="text">
    <string>Назад</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Закрыть приложение</string>
   </property>
  </action>
  <action name="settingsAppAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="aboutAccountAction">
   <property name="text">
    <string>Информация об аккаунте</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
